src/services/data-exports/export-cnfs-hub/repositories/export-cnfs-hub.repository.js

```diff
@@ -0,0 +1,90 @@
+const getStructureAndConseillerByDepartement = db => async departement => db.collection('structures').aggregate([
+  {
+    $match: {
+      codeDepartement: { $in: departement }
+    }
+  },
+  {
+    $lookup: {
+      localField: '_id',
+      from: 'conseillers',
+      foreignField: 'structureId',
+      as: 'conseiller',
+      pipeline: [
+        {
+          $match: {
+            statut: 'RECRUTE'
+          }
+        }
+      ]
+    }
+  },
+  { $unwind: '$conseiller' },
+  {
+    $project: {
+      'nom': 1,
+      'codeDepartement': 1,
+      'insee': 1,
+      'contact': 1,
+      'prenom': 1,
+      'conseiller.nom': 1,
+      'conseiller.prenom': 1,
+      'conseiller.emailCN': 1,
+      'conseiller.codePostal': 1,
+      'conseiller.codeRegion': 1
+    }
+  }
+]).toArray();
+
+const getStructureAndConseillerByDepartementHubAntillesGuyane = db => async departement => db.collection('structures').aggregate([
+  {
+    $match: {
+      $or: [
+        { codeDepartement: { $eq: departement[0] } },
+        { $and: [
+          { codeCom: { $eq: departement[1] } },
+          { codeDepartement: { $eq: '00' } }
+        ] },
```

Comment:
Faudrait pas se baser sur les index, si y a une entrée qui est ajoutée dans le hub, il faudra ajouter un [2]
En fait, le 978 est effectivement un cas particulier : tu peux laisser ton $or avec codeDepartement: avec un $in et laisser le $and avec le 978 en dur